apps/desktop-wallet/src/features/historicChart/HistoricWorthChart.tsx

```diff
@@ -46,11 +54,14 @@ const startingDates: Record<ChartLength, Dayjs> = {
 }
 
 const HistoricWorthChart = memo(function HistoricWorthChart({
+  isChartVisible,
+  isChartInitiallyHidden,
```

Comment:
I find those name confusing, it's like they're function returning a boolean.
Maybe we could use `shouldChartBeVisible` & `shouldChartBeInitiallyHidden`, or just `chartVisible` & `chartInitiallyHidden`.

Reply:
Fixed and pushed